From 18812e5cc5bd4cdc7cc7ef87a18bce78adbb4b13 Mon Sep 17 00:00:00 2001
From: Ilari Liusvaara <ilari.liusvaara@elisanet.fi>
Date: Fri, 11 Nov 2011 03:05:48 +0200
Subject: [PATCH 5/7] Fix unserialization of 64-bit signed integers

---
 nall/serializer.hpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/nall/serializer.hpp b/nall/serializer.hpp
index ff2337a..e6bc8fa 100755
--- a/nall/serializer.hpp
+++ b/nall/serializer.hpp
@@ -58,7 +58,7 @@ namespace nall {
         for(unsigned n = 0; n < size; n++) idata[isize++] = value >> (n << 3);
       } else if(imode == Load) {
         value = 0;
-        for(unsigned n = 0; n < size; n++) value |= idata[isize++] << (n << 3);
+        for(unsigned n = 0; n < size; n++) value |= (unsigned long long)idata[isize++] << (n << 3);
       } else if(imode == Size) {
         isize += size;
       }
-- 
1.7.9.48.g85da4d

