From e397bd46f17d6ea00c8c96d5a8e0c5f5b4a6f642 Mon Sep 17 00:00:00 2001
From: Ilari Liusvaara <ilari.liusvaara@elisanet.fi>
Date: Sun, 14 Oct 2012 23:31:36 +0300
Subject: [PATCH 10/27] Fix compiling on GCC 4.7

---
 nall/string.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nall/string.hpp b/nall/string.hpp
index 1b255ce2..07a64dfc 100755
--- a/nall/string.hpp
+++ b/nall/string.hpp
@@ -25,8 +25,8 @@
 #include <nall/string/base.hpp>
 #include <nall/string/bml.hpp>
 #include <nall/string/bsv.hpp>
-#include <nall/string/core.hpp>
 #include <nall/string/cast.hpp>
+#include <nall/string/core.hpp>
 #include <nall/string/compare.hpp>
 #include <nall/string/convert.hpp>
 #include <nall/string/cstring.hpp>
-- 
2.15.0.rc1

