From 36777ec11c093a1cc902b8b72188e06f49a4d0f2 Mon Sep 17 00:00:00 2001
From: Ilari Liusvaara <ilari.liusvaara@elisanet.fi>
Date: Sun, 14 Oct 2012 23:31:36 +0300
Subject: [PATCH] Fix compiling on GCC 4.7

---
 bsnes/nall/string.hpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bsnes/nall/string.hpp b/bsnes/nall/string.hpp
index 1b255ce..07a64df 100755
--- a/bsnes/nall/string.hpp
+++ b/bsnes/nall/string.hpp
@@ -25,8 +25,8 @@
 #include <nall/string/base.hpp>
 #include <nall/string/bml.hpp>
 #include <nall/string/bsv.hpp>
-#include <nall/string/core.hpp>
 #include <nall/string/cast.hpp>
+#include <nall/string/core.hpp>
 #include <nall/string/compare.hpp>
 #include <nall/string/convert.hpp>
 #include <nall/string/cstring.hpp>
-- 
1.7.10.4

